ISSUE_TRIAGE.md

```diff
@@ -29,6 +29,28 @@ The first step is in determining whether the issue is a bug, help question or fe
 4. A reproduction should be confirmed by at least one person other than the original reporter. Run the reproduction and validate that the bug exists; then make a note of your findings on the issue. If a reproduction is supplied but doesn't work, add the `can't-reproduce` label and make a comment describing what happened.
 5. Finally, once you've confirmed the reproduction add the `confirmed` label and [classify](#classification) the issue (removing the `can't-reproduce` label if it exists).
 
+#### Bug Issue Lifespan
```

Comment:
I'm feeling mighty nit-picky right now, so I'll lowercase the second and third words of this level-4 heading and keep only the first words capitalized in the subsequent headings.  (Mainly for consistency with the level 1 and 2 headings above, but also because these are nested further in the tree of this document (H3s and H4s).

Just FYI, I'll do this before merging.